charts/hivemq-edge/templates/license-secret.yaml

```
@@ -0,0 +1,12 @@
+{{- if .Values.license.create }}
+apiVersion: v1
+kind: Secret
+metadata:
+  labels:
+    {{- include "hivemq-edge.labels" . | nindent 4 }}
+  name: {{ include "hivemq-edge.name" (dict "prefix" "hivemq-edge" "name" "license" "releaseName" .Release.Name) }}
+  namespace: {{ .Release.Namespace }}
+type: Opaque
+data:
+  license.edgelic : {{ .Files.Get .Values.license.create.file | b64enc }}
```

Shouldn't this be the following instead?

```suggestion
  license.edgelic : {{ .Files.Get .Values.license.file | b64enc }}
```

---

Replaced by a different approach.